Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:jammy AS builder
+LABEL maintainer="TA <ta@lightscale.io>"
+
+RUN echo 'Etc/UTC' > /etc/timezone && \
+    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends tzdata && \
+    rm -rf /var/lib/apt/lists/*
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    ca-certificates \
+    curl  \
+    gnupg && \
+    rm -rf /var/lib/apt/lists/*
+RUN curl https://bazel.build/bazel-release.pub.gpg | \
+    apt-key add - && \
+    echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | \
+    tee /etc/apt/sources.list.d/bazel.list
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    bazel \
+    bazel-6.2.1 && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV HOME /root
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    git \
+    make \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    libncursesw5-dev \
+    xz-utils \
+    tk-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libffi-dev \
+    liblzma-dev \
+    python-is-python3 && \
+    rm -rf /var/lib/apt/lists/*
+RUN curl https://pyenv.run | bash
+RUN CONFIGURE_OPTS=--enable-shared $HOME/.pyenv/bin/pyenv install 3.10.12
```

Comment:
Could you define the python version 3.10.12 with https://docs.docker.com/reference/dockerfile/#arg?